lib/ide/src/controller/project.rs

```diff
@@ -78,47 +76,183 @@ impl Handle {
         let data = Controller::new_running(file_manager_transport);
         Self::new_from_data(data)
     }
+
+    /// Returns a text controller for given file path.
```

Comment:
I'd clarify that it works both for plain text files and modules.
Now I wondered if it was meant to say "plain text".